conf/messages.en

```diff
@@ -216,7 +216,7 @@ valuationMethod.heading = Which method do you want to use to value your goods?
 valuationMethod.caption = About the goods
 valuationMethod.paragraph.2 = We will then review your proposed method and provide a decision. Our decision will be legally binding.
 valuationMethod.paragraph.3 = We might contact you if we need more information or to recommend a more appropriate method.
-valuationMethod.link.text = work out the customs value of your imports (opens in new tab)
+valuationMethod.link.text = calculate the customs value of your imports (opens a new tab).
```

Comment:
If you are changing (opens in new tab) to (opens a new tab) then there is a discrepancy with all of the other links. We need to decide which one we are going for.